src/common/aes/aes.h

```diff
@@ -147,6 +149,32 @@ void OQS_AES256_CTR_inc_stream_iv(const uint8_t *iv, size_t iv_len, const void *
  */
 void OQS_AES256_CTR_inc_stream_blks(void *ctx, uint8_t *out, size_t out_blks);
 
+struct OQS_AES_callbacks {
```

Comment:
We aim to be verbose in comments on the public API, so could you add some comment about (the purpose) of this struct, please?

Reply:
Done.